llpc/test/shaderdb/partial_pipeline_cache_test/partial_pipeline_cache_test.ll

```diff
@@ -0,0 +1,11 @@
+; Test that partial pipeline cache is handled correctly.
+; BEGIN_SHADERTEST
+; RUN: amdllpc -spvgen-dir=%spvgendir% -v -shader-cache-mode=1 %S/pipelines/*.pipe | FileCheck -check-prefix=SHADERTEST %s
+; SHADERTEST: Non fragment shader cache miss. 
+; SHADERTEST-NEXT: Fragment shader cache miss.
+; SHADERTEST: Non fragment shader cache hit. 
+; SHADERTEST-NEXT: Fragment shader cache miss.
+; SHADERTEST: Non fragment shader cache miss. 
+; SHADERTEST-NEXT: Fragment shader cache hit.
```

Comment:
nit: could you align these? I mean something like:
```
; SHADERTEST:      Non fragment shader cache miss. 
; SHADERTEST-NEXT: Fragment shader cache miss.
; SHADERTEST:      Non fragment shader cache hit. 
```

Reply:
Fixed.